src/retry-failed-messages/src/lib/shouldRetry.test.ts

```diff
@@ -7,39 +7,49 @@ describe("shouldRetry", () => {
   it("should not retry new messages too early", () => {
     const receivedDate = new Date(Date.now() - 1799000)
     const message = mockAuditLog(receivedDate)
+    const event = mockAuditLogEvent("error", "Failed event", new Date(Date.now() - 1799000))
```

Comment:
`1799000` and `1800000` are both magic numbers - mind assigning these to variables with names?